src/components/CommentList.jsx

```diff
@@ -0,0 +1,23 @@
+import "../styling/ArticleList.css";
+import { useFetchComments } from "../hooks/useFetch";
+import CommentCard from "./CommentCard";
+
+function CommentList(props) {
+  const { article } = props;
+  const { commentList, isLoading } = useFetchComments(article);
+
+  if (isLoading) return <progress></progress>;
+  return (
+    <main>
+      <section>
+        <ul id="commentrList">
```

Comment:
```suggestion
    <main>
      <section>
        <ul id="commentrList">
```
If the comments list is the `main` section then there's no need for an additional `section` within it. If it's not the `main` section then you can remove both of them and just return a `ul` of comments 👍